presto-docs/src/main/sphinx/develop/secrets-manager.rst

```diff
@@ -0,0 +1,40 @@
+=====================
+Secrets Manager
+=====================
+
+Presto supports Custom Secrets Manager Plugins to retrieve secrets from secrets manager implementations of users' choice.
+By default, Hashicorp Vault is integrated as part of presto-vault module. More implementations can be plugged into the framework by extending
```

Comment:
```suggestion
By default, Hashicorp Vault is integrated as part of presto-vault module. More implementations can be plugged into the framework by extending
```
Perhaps "the presto-vault module", or "a presto-vault module", whichever you feel fits best. 

Does presto-vault exist in the codebase or URI-accessible such that a useful link exists for "presto-vault"?

Reply:
Once it is merged presto-vault will exist in the code base as readily useful plugin.

For example in my for it looks like this

https://github.com/skairali/presto/tree/vault/presto-vault

Detailed usage is documented here https://github.com/skairali/presto/tree/vault/presto-vault#vault-secrets-manager-plugin-for-presto

Reply:
Good enough for now. When presto-vault code is merged I ask that you come back to this and add an appropriate link in the documentation.